build.sbt

```diff
@@ -88,7 +88,7 @@ lazy val pekkoRuntime = (project in file("pekko-runtime"))
   .settings(commonSettings *)
   .dependsOn(core)
   .settings(
-    libraryDependencies ++= catsEffectStd ++ pekkoProvided ++ log4cats ++ scalapbCustomizations ++ (mUnit :+ pekkoTypedTestkit % pekkoVersion)
+    libraryDependencies ++= catsEffectStd ++ pekkoProvided ++ log4cats ++ scalapbCustomizations ++ (mUnit ++ logback ++ log4catsSlf4j :+ pekkoTypedTestkit % pekkoVersion)
```

Comment:
nit: I must miss why you need this log related extra depencendies for the pekko runtime, is not with the existing log4cats enough?

Reply:
yes thanks you're right, actually replacing log4cats-core by log4cats-slf4j